src/IncomingWebhook.ts

```diff
@@ -67,6 +70,10 @@ export class IncomingWebhook {
       return;
     });
```

Comment:
maybe a `catch()` to change errors into a `CodedError`?

Reply:
Oh, good *catch*! ([I apologize for that pun.](https://img.etsystatic.com/il/6637bd/749993407/il_570xN.749993407_65ss.jpg))

Reply:
![](https://media.giphy.com/media/198Ff4cvw8eVq/200.gif)